### PR TITLE
rfr-4 Add Settings Page

### PR DESCRIPTION
**Pull request by fin1989**

### CODE REVIEW

File: app/src/main/res/xml/preferences.xml [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+    <PreferenceCategory
+        android:key="refresh_category"
+        android:title="Automated Refresh">
+
+        <CheckBoxPreference
+            android:defaultValue="true"
+            android:key="perform_refresh"
+            android:summary="Enable or disable automatic refresh of all feeds"
+            android:title="Enabled" />
```
Comment (Vovchyk) [inline app/src/main/res/xml/preferences.xml]: string resource?

File: app/src/main/res/xml/preferences.xml [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+        <ListPreference
+            android:defaultValue="30"
+            android:dependency="perform_refresh"
+            android:entries="@array/refresh_interval"
+            android:entryValues="@array/refresh_interval_values"
+            android:key="refresh_interval"
+            android:summary="Interval between two automated refreshes"
+            android:title="Refresh interval" />
```
Comment (Vovchyk) [inline app/src/main/res/xml/preferences.xml]: same here..

File: app/src/main/res/layout/activity_settings.xml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:orientation="vertical" android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <LinearLayout
```
Comment (Vovchyk) [inline app/src/main/res/layout/activity_settings.xml]: probably one LinearLayout should be enough here (with id "content")

File: app/src/main/AndroidManifest.xml [hunk 4]
```diff
@@ -23,6 +23,10 @@
             </intent-filter>
         </activity>
 
+        <activity android:name=".ui.activities.SettingsActivity"
+            android:label="Settings">
```
Comment (fin1989) [inline app/src/main/AndroidManifest.xml]: @Vovchyk done

File: app/src/main/java/rss/feed/reader/ui/activities/SettingsActivity.java [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+/**
+ * Created by omartynets on 29.09.2016.
+ */
+public class SettingsActivity extends BaseActivity{
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_settings);
+
```
Comment (fin1989) [inline app/src/main/java/rss/feed/reader/ui/activities/SettingsActivity.java]: @Vovchyk done

File: app/src/main/res/menu/menu_main.xml [hunk 6]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <item
```
Comment (fin1989) [inline app/src/main/res/menu/menu_main.xml]: @Vovchyk done

File: app/src/main/res/xml/preferences.xml (hunk 1 above)
Comment (fin1989) [inline app/src/main/res/xml/preferences.xml]: @Vovchyk done

File: app/src/main/res/xml/preferences.xml (hunk 2 above)
Comment (fin1989) [inline app/src/main/res/xml/preferences.xml]: @Vovchyk done

File: app/src/main/res/layout/activity_settings.xml (hunk 3 above)
Comment (fin1989) [inline app/src/main/res/layout/activity_settings.xml]: @Vovchyk done

**Comment by fin1989** [comment]
@Vovchyk done

File: app/src/main/java/rss/feed/reader/ui/activities/SettingsActivity.java (hunk 5 above)
Comment (Vovchyk) [inline app/src/main/java/rss/feed/reader/ui/activities/SettingsActivity.java]: I meant after "R.xml.preferences,", but anyway)..

File: app/src/main/java/rss/feed/reader/ui/base/BaseActivity.java [hunk 7]
```diff
@@ -41,6 +45,24 @@ private AppComponent getAppComponent() {
...
+        MenuInflater inflater = getMenuInflater();
+        inflater.inflate(R.menu.menu_main, menu);
+        return true;
+    }
+
+    @Override
+    public boolean onOptionsItemSelected(MenuItem item) {
+        switch (item.getItemId()) {
+            case R.id.settings:
+                Navigation.toSettingsActivity(this);
```
Comment (qUser13) [inline app/src/main/java/rss/feed/reader/ui/base/BaseActivity.java]: @Vovchyk @fin1989 Should the view be responsible for navigation?

**Pull request merged**